ui/src/components/AdditionalInformationCard/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+type Props = {
+    additionalInformation: {
+        title: string;
+        url: string;
+        description?: string;
+    };
+};
+
+const AdditionalInformationCard: React.FC<Props> = (props): React.ReactElement => (
```

Comment:
Confused what React.ReactElement is doing here. Could also destructure the props

`const AdditionalInfoCard: React.FC<Props> = ({ additionalInformation: { title, url, description }}) => {`

Reply:
The React.ReactElement is just to specify the return type of the function component for typescript. I've destructured the props now too